src/middlewared/middlewared/plugins/vm/vm_lifecycle.py

```diff
@@ -106,6 +109,46 @@ def restart(self, job, id):
 
         self.start(id, {'overcommit': True})
 
+    @item_method
+    @accepts(Int('id'))
+    @returns()
+    def suspend(self, id):
```

Comment:
Are "suspend" and "pause" the same thing?

Reply:
Yes, they are the same thing